cvmfs/publish/repository.h

```diff
@@ -312,15 +314,13 @@ class __attribute__((visibility("default"))) Publisher : public Repository {
   void Migrate();
 
   const SettingsPublisher &settings() const { return settings_; }
-  bool in_transaction() const { return in_transaction_; }
-  bool is_publishing() const { return is_publishing_; }
+  const ServerFlagFile &in_transaction() const { return in_transaction_; }
```

Comment:
```suggestion
  const ServerFlagFile& in_transaction() const { return in_transaction_; }
```

Reply:
I chose `const Thing &method()` rather than `const Thing& method()` for consistency with the surrounding lines: https://github.com/cvmfs/cvmfs/blob/78b6e377b5ec65ee1714f21fd3c68cd4abc7f7f0/cvmfs/publish/repository.h#L316 and https://github.com/cvmfs/cvmfs/blob/78b6e377b5ec65ee1714f21fd3c68cd4abc7f7f0/cvmfs/publish/repository.h#L319-L321

If you prefer, I can insert a preliminary commit that reverses the spacing on those existing lines, and then match consistency with that?

Reply:
I didn't realise there were these instances in this code base, I actually though the cosmetic checker would complain without these changes.

We can go with your version, to be consistent.